ROADMAP.md

```diff
@@ -0,0 +1,32 @@
+# Roadmap
+
+- [Raw devices Implement](#Raw-devices-Implement)
+- [Manage volume group that user created](#Manage-volume-group-that-user-created)
+- [Data Migration and Backup](#Data-Migration-and-Backup)
+- [Nodes and devices selector](#Node-and-devices-selector)
+
+###Raw devices Implement
+
+Not only lvm, operator could orchestrate csi driver that support raw devices. we should implement csi driver.
+
+###Manage volume group that user created
+
+volume group that created by user has available storage capacity, use could config operator that could manage these volume groups.
+
+###Data Migration and Backup
+
+Make cronjob that do regular backup
+
+###Node and devices selector
+
+Enrich node and device selectors that makes deployment more flexible
+
```

Comment:
a lot of empty lines below.  We should remove them.

Reply:
> I don't know if I made it clear to you. Please feel free to contact me if you have any further questions.

Yes for sure, it is completely clear, @little-guy-lxr . I just commented if it is worth to have included these changes in the roadmap.